basic-visit-java/src/main/java/ai/vespa/example/visit/Visit.java

```diff
@@ -0,0 +1,55 @@
+package ai.vespa.example.visit;
+
+import com.yahoo.document.Document;
+import com.yahoo.document.DocumentId;
+import com.yahoo.documentapi.DocumentAccess;
+import com.yahoo.documentapi.DumpVisitorDataHandler;
+import com.yahoo.documentapi.ProgressToken;
+import com.yahoo.documentapi.VisitorControlHandler;
+import com.yahoo.documentapi.VisitorParameters;
+import com.yahoo.documentapi.VisitorSession;
+
+import java.util.concurrent.TimeoutException;
+
+public class Visit {
+
+    public static void main(String[] args) throws Exception {
+        VisitorParameters params = new VisitorParameters("true");
+        params.setLocalDataHandler(new DumpVisitorDataHandler() {
+
+            @Override
+            public void onDocument(Document doc, long timeStamp) {
+                System.out.print(doc.toXML(""));
```

Comment:
Vespa Team is deprecating XML interfaces and JSON is more future proof, so toJson is probably better